six/modules/c++/six.sidd/tests/test_compare_sidd.cpp

```diff
@@ -0,0 +1,208 @@
+#include <import/cli.h>
```

Comment:
This is just a pet peeve, but I hate these import headers.  I guess you don't have to, but bonus points if you change this to just bring in what you need from each module.